### PR TITLE
Feat(kclvm-win): Before calling the python version of kclvm in kcl.go check if kclvm is installed.

### PR DESCRIPTION
**Pull request by zong-zhe**
#### 1. Does this PR affect any open issues?(Y/N) and add issue references (e.g. "fix #123", "re #123".):

- [ ] N
- [x] Y 

issue #357 

#### 2. What is the scope of this PR (e.g. component or file name):

kclvm windows.

#### 3. Provide a description of the PR(e.g. more details, effects, motivations or doc link):

- [x] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Other

Before calling the python version of kclvm in kcl.go check if kclvm is installed. If it is not installed, call pip to install it.

#### 4. Are there any breaking changes?(Y/N) and describe the breaking changes(e.g. more details, motivations or doc link):

- [x] N
- [ ] Y 

#### 5. Are there test cases for these changes?(Y/N) select and add more details, references or doc links:

- [ ] Unit test
- [ ] Integration test
- [ ] Benchmark (add benchmark stats below)
- [ ] Manual test (add detailed scripts or steps below)
- [x] Other

#### 6. Release note

Please refer to [Release Notes Language Style Guide](https://kusionstack.io/docs/governance/release-policy/) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by Peefy — DISMISSED** [review]
LGTM

File: scripts/build-windows/kcl.go [hunk 1]
```diff
@@ -6,14 +6,26 @@
...
 	"unsafe"
 )
 
 func main() {
 	// kclvm -m kclvm ...
+	inputPath := os.Args[0]
+
+	if _, err := os.Stat(inputPath); os.IsNotExist(err) {
+		fmt.Println("Input path does not exist")
+		return
```
Comment (chai2010) [inline scripts/build-windows/kcl.go:23]: `os.Exit(1)`

File: scripts/build-windows/kcl.go [hunk 2]
```diff
@@ -6,14 +6,26 @@
...
 	"os"
+	"os/exec"
 	"path/filepath"
 	"syscall"
 	"unsafe"
 )
 
 func main() {
 	// kclvm -m kclvm ...
+	inputPath := os.Args[0]
```
Comment (chai2010) [inline scripts/build-windows/kcl.go:19]: `os. Executable()` ?

File: scripts/build-windows/kcl.go [hunk 3]
```diff
@@ -57,3 +69,37 @@ func findKclvm_dllPath() string {
...
 	return kclvmName
 }
+
+func install_kclvm(installed_path string) {
+	// Check if Python3 is installed
+	cmd := exec.Command("cmd", "/C", "where python3")
+	_, err := cmd.Output()
+	if err != nil {
+		fmt.Println("Python3 is not installed, details: ", err)
+		return
```
Comment (chai2010) [inline scripts/build-windows/kcl.go:79]: `os.Exit(1)` ?

File: scripts/build-windows/kcl-doc.go [hunk 4]
```diff
@@ -40,7 +40,7 @@ func findKclvm_dllPath() string {
 	kclvmName := "python39.dll"
 
 	if exePath, _ := os.Executable(); exePath != "" {
-		exeDir := filepath.Dir(exePath)
+		exeDir := filepath.Join(exePath, "../..")
```
[the remark this answers is not in the document]
Comment (zong-zhe) [reply scripts/build-windows/kcl-doc.go]: fixed.

File: scripts/build-windows/kcl-fmt.go [hunk 5]
```diff
@@ -40,7 +40,7 @@ func findKclvm_dllPath() string {
 	kclvmName := "python39.dll"
 
 	if exePath, _ := os.Executable(); exePath != "" {
-		exeDir := filepath.Dir(exePath)
+		exeDir := filepath.Join(exePath, "../..")
```
[the remark this answers is not in the document]
Comment (zong-zhe) [reply scripts/build-windows/kcl-fmt.go]: fixed.

File: scripts/build-windows/kcl.go (hunk 1 above)
Comment (zong-zhe) [reply scripts/build-windows/kcl.go]: fixed.

File: scripts/build-windows/kcl.go (hunk 2 above)
Comment (zong-zhe) [reply scripts/build-windows/kcl.go]: fixed.

File: scripts/build-windows/kcl.go (hunk 3 above)
Comment (zong-zhe) [reply scripts/build-windows/kcl.go]: fixed.

**Review by Peefy — DISMISSED** [review]
LGTM

**Review by chai2010 — APPROVED** [review]
LGTM

**Review by Peefy — APPROVED** [review]
LGTM

**Pull request merged**